tests/optional_kernel_features/kernel_features_device_has_exceptions.cpp

```diff
@@ -0,0 +1,330 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for the exception that are thrown by [[sycl::device_has]]
+//  attribute.
+//
+*******************************************************************************/
+
+#include "../common/disabled_for_test_case.h"
+#include "catch2/catch_template_test_macros.hpp"
+#include "kernel_features_common.h"
```

Comment:
```suggestion
// FIXME: re-enable compilation with hipSYCL when `sycl::errc` is supported
#ifndef __HIPSYCL__
#include "kernel_features_common.h"
#endif
```

Reply:
It will work. Applied this in https://github.com/KhronosGroup/SYCL-CTS/pull/287/commits/85c27137801e86d89a7e9f3eec2fd1d7524716da